tensorboard/webapp/widgets/line_chart_v2/lib/renderer/renderer_types.ts

```diff
@@ -84,6 +84,11 @@ export interface ObjectRenderer<CacheValue = {}> {
   destroyObject(cachedValue: CacheValue): void;
 
   setUseDarkMode(useDarkMode: boolean): void;
+
+  /**
+   * Disposes rendering context. After invocation, renderer will never be used.
```

Comment:
"will never be used" or "should never be used"?

Reply:
This is indicating to the implementer of renderer that one should expect that the renderer will no longer be used ever after `dispose` is called. `should never be used` sounds like an instruction to a user of the renderer. :thinking: 

I could see this jsdoc being used for both cases. Adopted your phrasing instead.